File: 1041.cs
using System;

class Program {
  public static void Main (string[] args) {
    string []XY = Console.ReadLine().Split(' ');

    double X = Convert.ToDouble(XY[0]);
    double Y = Convert.ToDouble(XY[1]);

    if(X > 0 && Y > 0)
      Console.WriteLine("Q1");

    if(X < 0 && Y > 0)
      Console.WriteLine("Q2");
    
    if(X < 0 && Y < 0)
      Console.WriteLine("Q3");

    if(X > 0 && Y < 0)
      Console.WriteLine("Q4");

    if(Y == 0 && X > 0 || Y == 0 && 0 > X)
      Console.WriteLine("Eixo X");

    if(X == 0 && Y > 0 || X == 0 && 0 > Y)
      Console.WriteLine("Eixo Y");
    
    if(X == 0 && Y == 0)
      Console.WriteLine("Origem");
    
  }
}

File: 1041.cpp

#include <stdlib.h>
#include <stdio.h>

int main() {

    float x, y;
    scanf("%f %f", &x, &y);

    if (x > 0)
    {
      if (y > 0)
      {
        printf("Q1\n");
      }

      if (y < 0)
      {
        printf("Q4\n");
      }

      if (y == 0)
      {
        printf("Eixo X\n");
      }
      
    }

    if (x < 0)
    {
      if (y > 0)
      {
        printf("Q2\n");
      }

      if (y < 0)
      {
        printf("Q3\n");
      }

      if (y == 0)
      {
        printf("Eixo X\n");
      }

    }
    
    if (x == 0)
    {
      if (y == 0)
      {
        printf("Origem\n");
      }

      else
      {
        printf("Eixo Y\n");
      }
    }
   

     return 0;
}